.gitignore

```diff
@@ -0,0 +1 @@
+*.swp
```

Comment:
This isn't really project-specific but instead depends on the software you choose to use and thus should go into `~/.config/git/ignore` IMO.